nym-vpn-app/src-tauri/bundle/windows/installer.nsi

```diff
@@ -192,7 +192,8 @@ FunctionEnd
         DetailPrint "vpnd version is pre 1.14, using --uninstall flag"
         ExecWait '"$INSTDIR\nym-vpnd.exe" --uninstall' $9
       ${Else}
-        ExecWait '"$INSTDIR\nym-vpnd.exe" uninstall-service' $9
+        DetailPrint "Waiting 30 seconds for the vpnd service to stop and uninstall"
+        ExecWait '"$INSTDIR\nym-vpnd.exe" uninstall-service --wait 30' $9
```

Comment:
I am not sure if this will break any existing installations, but let's keep the duration of `--wait` baked in the daemon.

Reply:
Yes it will!  Reverted.